rest-e2e-tests/types.ts

```diff
@@ -0,0 +1,4 @@
+interface EthereumAccount { 
```

Comment:
Let's have one type per file. Ie. rename to `EthereumAccount.ts`

Reply:
Fixed